### PR TITLE
Update sec policy

### PR DESCRIPTION
**Pull request by johrstrom**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
+We hope to have more audits on a regular ongoing basis and to share those results when applicable.
 
 ## Reporting a Vulnerability
 
-Per our project website: http://openondemand.org/, if you have security concerns or think you have found a vulnerability in Open OnDemand, please contact us directly via email on the news list.  You can access the news list at https://lists.osu.edu/mailman/listinfo/ood-users or directly via ood-users@lists.osc.edu.  Emails sent to it are only seen by the core project team.
+
+If you have security concerns or think you have found a vulnerability in Open OnDemand,
+please contact us directly via [email](mailto:ood-users@lists.osc.edu) on the news list found [here](https://lists.osu.edu/mailman/listinfo/ood-users).
+
+Your email will be answered promptly by one of our team.
```
Comment (ericfranz) [inline SECURITY.md]: Add back this sentence:

>Emails sent to it are only seen by the core project team.

That explains the privacy which is important. After adding that back this sentence becomes redundant and can be removed:

>Your email will be answered promptly by one of our team.

File: SECURITY.md [hunk 2]
```diff
@@ -1,9 +1,38 @@
...
 ## Security Audits
 
-[Trusted CI](https://trustedci.org/) , the NSF Cybersecurity Center of Excellence conducted an in-depth vulnerability assessment of Open OnDemand, completing it in December 2018. This assessment included a careful review of the code, increasing our confidence in its security. The Ohio Supercomputing Center addressed the implementation issues (bugs) that were found during this review, producing a more robust revision of Open OnDemand.
+We have used [Trusted CI](https://trustedci.org/) in the past, the NSF Cybersecurity Center of 
+Excellence conducted an in-depth vulnerability assessment of Open OnDemand, completing 
+it in December 2018. This assessment included a careful review of the code, increasing 
+our confidence in its security. The Ohio Supercomputing Center addressed the implementation 
+issues (bugs) that were found during this review, producing a more robust revision of Open OnDemand.
+
+We hope to have more audits on a regular ongoing basis and to share those results when applicable.
```
Comment (ericfranz) [inline SECURITY.md]: Revert this back to the original copy Alan added and remove this line:

>We hope to have more audits on a regular ongoing basis and to share those results when applicable

File: SECURITY.md [hunk 3]
```diff
@@ -1,9 +1,38 @@
...
+
+  * Confirm the problem and determine the affected versions.
+  * Audit code to find any potential similar problems.
+  * Prepare fixes for all releases still under maintenance. These fixes will be
+    released as fast as possible.
+
+## Comments on this Policy
+
+If you have suggestions on how this process could be improved please submit 
+a ticket, open a [discorse](https://discourse.osc.edu/) topic or open a pull request.
```
Comment (ericfranz) [inline SECURITY.md]: Discourse should be capitalized.

File: SECURITY.md (hunk 2 above)
Comment (johrstrom) [reply SECURITY.md]: You don't want audits to become regular or results public?

Comment (johrstrom) [reply SECURITY.md]: Also, Alan's line is one long line, this is broken up to be more readable in an ide/viewer.  You know, 100 column limit sort of thing.

File: SECURITY.md (hunk 1 above)
Comment (johrstrom) [reply SECURITY.md]: Done.

File: SECURITY.md (hunk 3 above)
Comment (johrstrom) [reply SECURITY.md]: Done.

**Comment by achalker** [comment]
Sure I’d like them to be regular and public, but I don’t know we have the budget for that nor the ability to publicize all the results.


-------------------------
Alan Chalker, Ph.D.
alanc@osc.edu<mailto:alanc@osc.edu>
614-247-8672

**Comment by johrstrom** [comment]
OK, removed, but to be fair I had 2 qualifiers being `We hope to have ...` and `share ... when applicable`.

**Pull request merged**